.github/config/nightly.yaml

```diff
@@ -24,6 +24,7 @@ flavors:
         build_image_qemu: true
         build_image_vbox: true
         build_raw_image: true
+        luet_script_override: ""
```

Comment:
so the value is false or empty?

Reply:
I just changed it slightly, it is a string that you can use to specify a luet version to use directly :+1: